nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedRecipe.groovy

```diff
@@ -46,6 +47,8 @@ class ConanHostedRecipe
 {
   public static final String NAME = 'conan-hosted'
 
+  public static final String HOSTED_ENABLED_PROPERTY = 'nexus.conan.hosted.enabled'
```

Comment:
If this constant used only in this file in tests make sense to make it private and mark VisibleForTesting annotation?

Reply:
Yes, we can use VisibleForTesting but, it is just constant, so, for what do we need to hide that? I think this case does not need VisibleForTesting